services/pagerduty_alerts.rb

```diff
@@ -55,10 +55,11 @@ def process_incs(incs)
 
     for inc in incs
       redis_key = "#{REDIS_KEY_PREFIX}:#{inc['id']}"
-      Alert.create(redis_key, { 
+      Alert.create(redis_key, {
         message: {
           service: inc['service']['name'],
-          description: inc['trigger_summary_data']
+          description: inc['trigger_summary_data'],
+          acknowledged: inc['status']=="acknowledged"?true:false
```

Comment:
I don't htink acknowledged is part of the message - it's metadata - so can you move it up a level  to live alongside message please?